### PR TITLE
Setup Instruction for Developement Env

### PR DESCRIPTION
**Pull request by aj-abhishek-jadav**
Changes made in Readme file

### CODE REVIEW

**Comment by aj-abhishek-jadav** [comment]
hi @excid3 @mybuddyandrew, Have added instructions on how to setup project locally. request you to please Review.

**Comment by AhmadR-Coder-max** [comment]
> hi @excid3 @mybuddyandrew, Have added instructions on how to setup project locally. request you to please Review.

Hi @AbhishekJadav1998!
I had a chance to review your changes and noticed a few things that could be improved. 

Would it be possible to include some additional steps for forking this repository and related actions?

I think it'd be more beneficial to provide the versions of Ruby and Rails that this project is using. So we can get rid of any [installation commands](https://github.com/excid3/beginnerbounties.com/pull/27/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R10)

Thanks!

**Comment by aj-abhishek-jadav** [comment]
> > hi @excid3 @mybuddyandrew, Have added instructions on how to setup project locally. request you to please Review.
> 
> Hi @AbhishekJadav1998! I had a chance to review your changes and noticed a few things that could be improved.
> 
> Would it be possible to include some additional steps for forking this repository and related actions?
> 
> I think it'd be more beneficial to provide the versions of Ruby and Rails that this project is using. So we can get rid of any [installation commands](https://github.com/excid3/beginnerbounties.com/pull/27/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R10)
> 
> Thanks!

Hi @AhmadR-Coder-max
Github have awesome documentation on how forking works and how we can fork a repository. Hence added link.
Regarding providing ROR version, i think `bundle install` will handle versions automatically. anyway that command is prerequisite for setting up dev env

File: README.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+3. Install all dependencies with bundle
+
+        $ bundle install
+
+4. Setup Database and run migrations
+
+        $ rake db:create
+        $ rake db:migrate --trace
+
+5. Precompile assets
```
Comment (excid3) [inline README.md]: You should not precompile assets in development. This is only needed for production.

Comment (aj-abhishek-jadav) [reply README.md]: ohh ohk. I thought assets precompile is needed for every environment. 
I've updated the PR, removed precompile step.

Thank you for your valuable time in reviewing, it really means a lot 😍 to me @excid3

**Comment by Kunj-Choksi** [comment]
Didn't know someone was already working, I made my version for this. [gist](https://gist.github.com/Kunj-Choksi/3888ab3a207905b2b3f46255d01ec4e5)
Feel free to use any relevant information if you want.

**Comment by aj-abhishek-jadav** [comment]
Hey @excid3 can you please review this PR. If any changes needed, please let me know.

Thanks.

**Comment by aj-abhishek-jadav** [comment]
@excid3 Can you please Review/Merge this?

**Pull request merged**